modules/ROOT/pages/design-migrating-ms.adoc

```diff
@@ -0,0 +1,132 @@
+
+== What is the Mocking Service
+
+The mocking service lets you test endpoints that are defined in an API specification. The advantage to using the mocking service is that you do not need to implement the API before testing. To test an endpoint, the mocking service simulates calls to its methods.
+
+For example, when you are designing an API specification in API Designer, you can simulate a call to an endpoint’s GET method to see whether the response body for the 200 status code is as you want it. As a further example, if you are evaluating whether you want to implement an API specification that is in Exchange, you can simulate a call to an endpoint’s GET method to find out what the response body looks like.
+
+
+== How to Migrate an API-Specification Project to Version 2 of the Mocking Service
+
+If the mocking service was switched on for an API-specification project before January 10, 2019 and has been running continuously since then, follow these steps to migrate the project to version 2 of the mocking service:
+
+. In the notification that appears in the top-right corner after you open the project, click *Upgrade*.
++
+*Result:* The mocking service is switched off and then on. The form of the base URI that the mocking service generates now differs from the form of the previous base URI.
++
+Version 1 of the mocking service used base URIs of the form `+https://mocksvc.mulesoft.com/mocks/{ID}+`.
++
+Version 2 uses base URIs of the form `+https://anypoint.mulesoft.com/mocking/api/v1/links/{new-ID}+`.
++
+For example, the URI for a GET method for an endpoint named `/customers` in a particular specification might have looked like this with version 1 of the mocking service:
++
+`+https://mocksvc.mulesoft.com/mocks/7d19e624-a0d6-4d77-9fb8-8c78a8aeb575/api/customers+`
++
+Version 2 of the mocking service changes the URI to this and uses a different ID:
++
+`+https://anypoint.mulesoft.com/mocking/api/v1/links/62a7d6af-ebc7-43fa-a815-e415e329f2ff/api/customers+`
+
+. If any third-party tools are testing endpoints in the API specification for that project, replace the previous base URI with the new base URI in the links that those tools are using.
++
+For example, suppose that this URI is being used in a tool:
++
+`+https://mocksvc.mulesoft.com/mocks/7d19e624-a0d6-4d77-9fb8-8c78a8aeb575/api/customers+`
++
+You would need to replace `+https://mocksvc.mulesoft.com/mocks/7d19e624-a0d6-4d77-9fb8-8c78a8aeb575+` with `+https://anypoint.mulesoft.com/mocking/api/v1/links/{new-ID}+`, where `new-ID` is the ID that version 2 of the mocking service generates.
++
+You can get the new base URI from version 2 of the mocking service after step 1 above.
+
+== New Features in Version 2 That You Can Use in API Designer
+
+Version 2 of the mocking service, which was released on January 10, 2019, introduced the following features that you can use in API Designer.
```

Comment:
How about...:

After you migrate your mocking service from Version 1 to Version 2, you test OAS endpoints, add behavioral headers, enjoy improved performance, and achieve access authentication in Exchange.

I think it is important that our doc going forward follow a plan by which the text following a Level-1 heading include a statement that summarizes the Level-2 content; that a Level-2 intro text likewise summarize the Level-3 content; and so on. A user-journey roadmap, if you will, that uses signposting so that users can decide if the topic is for them or not.